crowbar_framework/app/models/api/storage.rb

```diff
@@ -15,6 +15,43 @@
 #
 
 module Api
-  class Storages
+  class Storage < Tableless
+    def repocheck
+      {}.tap do |ret|
+        [:ceph, :system].each do |r|
+          ret[r] = {
+            available: false
+          }
+        end
+
+        # check for the system repos
+        ["sles12-sp2-pool", "sles12-sp2-updates"].each do |repo|
```

Comment:
you mean adding metadata to the yaml file, or taking something existing?
btw I moved this code to https://github.com/crowbar/crowbar-core/pull/633
lets continue there, as this is not the right repo